tap_intacct/discover.py

```diff
@@ -20,4 +20,7 @@ def load_metadata(schema):
         if field_name == "RECORDNO":
             mdata = metadata.write(mdata, (), 'table-key-properties', "RECORDNO")
 
+    # Since it reads CSV files from S3 individually hence FULL_TABLE replication
+    mdata = metadata.write(mdata, (), 'forced-replication-method', 'FULL_TABLE')
```

Comment:
Please re-verify the implementation. We are writing the bookmark [here](https://github.com/singer-io/tap-intacct/blob/master/tap_intacct/sync.py#L28), indicates that this should be incremental replication method.